pom.xml

```diff
@@ -1,76 +1,37 @@
 <?xml version="1.0"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <groupId>com.relayrides</groupId>
-    <artifactId>pushy</artifactId>
+    <artifactId>pushy-parent</artifactId>
+    <packaging>pom</packaging>
     <version>0.7.2-SNAPSHOT</version>
-    <name>Pushy</name>
+    <name>Pushy parent</name>
     <description>A Java library for sending push notifications</description>
+
     <parent>
         <groupId>org.sonatype.oss</groupId>
         <artifactId>oss-parent</artifactId>
         <version>7</version>
     </parent>
+
+    <modules>
+        <module>pushy</module>
```

Comment:
Actually, I think I changed my mind here if only for the petty reason that having everything start with `pushy-` was making autocomplete slightly annoying ;)

Besides—we can always change it later.